jwql/instrument_monitors/nirspec_monitors/ta_monitors/wata_monitor.py

```diff
@@ -53,8 +54,7 @@
 from jwql.utils import monitor_utils
 from jwql.utils.constants import JWST_INSTRUMENT_NAMES_MIXEDCASE
 from jwql.database.database_interface import session, engine
-from jwql.database.database_interface import NIRSpecTAQueryHistory, NIRSpecTAStats
-from jwql.utils.utils import ensure_dir_exists, filesystem_path, get_config, filename_parser
+from jwql.utils.utils import ensure_dir_exists, filesystem_path, get_config
 
 
 class WATA():
```

Comment:
This file also has some conflicts with changes on develop. I can help resolve them.